projects/packages/forms/src/blocks/option/edit.js

```diff
@@ -42,6 +42,15 @@ const OptionEdit = ( { attributes, clientId, context, name, setAttributes } ) =>
 	const useEnterRef = useEnter( { content: label, clientId, isStandalone } );
 	const useEnterRequiredRef = useEnter( { content: label, clientId, isStandalone } );
 
+	const isPreviewMode = useSelect( select => {
+		return select( blockEditorStore ).getSettings().isPreviewMode;
+	}, [] );
+	const placeholder = __( 'Add label…', 'jetpack-forms' );
+	const emptyToNull = str => ( str === '' ? null : str );
```

Comment:
I think this function could be moved out of the context its in so that it's not inline.

The same applies to the function of the same name in the label.